rad/src/reducers/mod.rs

```diff
@@ -125,29 +124,15 @@ mod tests {
     }
 
     #[test]
-    fn test_reduce_average_median_tapi_activation() {
```

Comment:
Maybe it can be useful to keep a test that checks tapi activation? By doing something like:

```rust
active_wips
            .active_wips
            .remove(&"WIP0017-0018-0019".to_string());
```

Although I think this is already tested in the validations crate.